dallinger/deployment.py

```diff
@@ -133,7 +133,6 @@ def deploy_sandbox_shared_setup(
     team = config.get("heroku_team", None)
     heroku_app = HerokuApp(dallinger_uid=heroku_app_id, output=out, team=team)
     heroku_app.bootstrap()
-    heroku_app.buildpack("https://github.com/stomita/heroku-buildpack-phantomjs")
```

Comment:
I thought we were going to wait on this change(?)

Reply:
Right! Thanks for reminding me. It slipped.